nflgame/game.py

```diff
@@ -355,6 +355,13 @@ def __init__(self, eid=None, fpath=None, **kwargs):
         self.gcJsonAvailable = self.data['gcJsonAvailable']
 
         if(self.data['gcJsonAvailable']):
+            # DEBUGGING
+            print('home {}'.format(self.home))
+            print(self.data['home']['stats']['team'])
+            print('away {}'.format(self.away))
+            print(self.data['away']['stats']['team'])
+
+
```

Comment:
Looks like you left some debugging statements in the PR. Probably should remove them?

Reply:
yeah -- sorry, I've been doing some separate debugging on `master`. I'll cut it out.